chart/compass/charts/director/static-groups.yaml

```diff
@@ -1,41 +0,0 @@
-{{- range $name := regexSplit "," .Values.adminGroupNames -1 }}
-- groupname: "{{ trim $name }}"
-  scopes:
-    - "application:read"
-    - "application:write"
-    - "application_template:read"
-    - "application_template:write"
-    - "integration_system:read"
-    - "integration_system:write"
-    - "runtime:read"
-    - "runtime:write"
-    - "label_definition:read"
-    - "label_definition:write"
-    - "eventing:manage"
-    - "tenant:read"
-    - "automatic_scenario_assignment:read"
-    - "automatic_scenario_assignment:write"
-    - "application.auths:read"
-    - "application.webhooks:read"
-    - "application_template.webhooks:read"
-    - "bundle.instance_auths:read"
-    - "document.fetch_request:read"
-    - "event_spec.fetch_request:read"
-    - "api_spec.fetch_request:read"
-    - "integration_system.auths:read"
-    - "runtime.auths:read"
-    - "fetch-request.auth:read"
-    - "webhooks.auth:read"
-    - "formation:write"
-    - "internal_visibility:read"
```

Comment:
This scope is currently missing from the hydrator's file. Maybe we should add it there as well?